app/templates/notifications/all.hbs

```diff
@@ -1,25 +1,30 @@
-{{#each model as |notification|}}
-  <div class="ui segment {{unless notification.isRead 'blue'}}">
-    <div class="ui grid">
-      <div class="row">
-        <div class="eight wide column">
-          <h4 class="ui header">{{notification.title}}</h4>
-          <p>{{sanitize notification.message}}</p>
+{{#if model}}
```

Comment:
Instead of this, use `each-else` -> https://guides.emberjs.com/v2.15.0/templates/displaying-a-list-of-items/#toc_empty-lists